01_fizz_buzz/tests/FizzBuzzTest.php

```diff
@@ -12,4 +14,44 @@ public function test_the_truth(): void
     {
         $this->assertTrue(true);
     }
+
+    #[DataProvider('ValidationProvider')]
+    public function test_is_divisible_test(
+        string $expected,
+        int $value
+    ) : void {
+        $fizzBuzz = new FizzBuzz();
+        $this->assertEquals($expected, $fizzBuzz->FizzBuzz($value));
+    }
+
+
+    public static function validationProvider(): array
+    {
+        return [
+            'not divisible' => [
```

Comment:
This is a bug in your system. What does the brief say we should return if the number does not meet any of the `Fizz`, `Buzz` or `Bar` conditions